plugins/inputs/sql/README.md

```diff
@@ -0,0 +1,192 @@
+		"Server=mssqlserver2.my.lan;Port=1433;User Id=telegraf;Password=secret;app name=telegraf"
+	]
+	hosts=["mssqlserver_cluster_1","mssqlserver_cluster_2"]
+
+	[[inputs.sql.query]]
+		measurement = "os_performance_counters"
+		ignore_other_fields=true
+		sanitize=true
+		query="SELECT * FROM sys.dm_os_performance_counters WHERE object_name NOT LIKE '%Deprecated%' ORDER BY counter_name"
+		tag_cols=["instance_name","object_name"]
+		field_name = "counter_name"
+		field_value = "cntr_value"
+```
+### Result:
+```
+> os_performance_counters,host=mssqlserver_cluster_1,object_name=MSSQL$TESTSQL2014:Broker_Statistics Activation_Errors_Total=0i 1494496261000000000
+> os_performance_counters,host=mssqlserver_cluster_1,object_name=MSSQL$TESTSQL2014:Cursor_Manager_by_Type,instance_name=TSQL_Local_Cursor Active_cursors=0i 1494496261000000000
+> os_performance_counters,instance_name=TSQL_Global_Cursor,host=mssqlserver_cluster_1,object_name=MSSQL$TESTSQL2014:Cursor_Manager_by_Type Active_cursors=0i 1494496261000000000
+> os_performance_counters,host=mssqlserver_cluster_1,object_name=MSSQL$TESTSQL2014:Cursor_Manager_by_Type,instance_name=API_Cursor Active_cursors=0i 1494496261000000000
+> os_performance_counters,host=mssqlserver_cluster_1,object_name=MSSQL$TESTSQL2014:Cursor_Manager_by_Type,instance_name=_Total Active_cursors=0i 1494496261000000000
+...
+
+```
+## Example for collect multiple counters defined as ROWS in a table (horizontal counter structure):
+Here we read multiple counters defined on same row where the counter name is the name of his column.
+In this example we force some counters datatypes: "MEMBERS","FIRST_CHANGE#" as integer, "BYTES" as float, "FIRST_TIME" as time. The field "UNIT" is used with the automatic driver datatype conversion.
+The column "ARCHIVED" is ignored
+
+###Config
+```
+[[inputs.sql]]
+	interval = "20s"
+
+	driver = "oci8"
+	keep_connection=true
+	servers  = ["telegraf/monitor@10.62.6.1:1522/tunapit"]
+	hosts=["oraclehost.my.lan"]
+	## Queries to perform
+	[[inputs.sql.query]]
+		query="select GROUP#,MEMBERS,STATUS,FIRST_TIME,FIRST_CHANGE#,BYTES,ARCHIVED from v$log"
+		measurement="log"
+		tag_cols=["GROUP#","STATUS","NAME"]
+		field_cols=["UNIT"]
+		int_fields=["MEMBERS","FIRST_CHANGE#"]
+		float_fields=["BYTES"]
+		time_fields=["FIRST_TIME"]
+		ignore_other_fields=true
+```
+### Result:
+```
+> log,host=pbzasplx001.wp.lan,GROUP#=1,STATUS=INACTIVE MEMBERS=1i,FIRST_TIME="2017-05-10 22:08:38 +0200 CEST",FIRST_CHANGE#=368234811i,BYTES=52428800 1494496874000000000
+> log,host=pbzasplx001.wp.lan,GROUP#=2,STATUS=CURRENT MEMBERS=1i,FIRST_TIME="2017-05-10 22:08:38 +0200 CEST",FIRST_CHANGE#=368234816i,BYTES=52428800 1494496874000000000
+> log,host=pbzasplx001.wp.lan,GROUP#=3,STATUS=INACTIVE MEMBERS=1i,FIRST_TIME="2017-05-10 16:00:55 +0200 CEST",FIRST_CHANGE#=368220858i,BYTES=52428800 1494496874000000000
+
+
+```
+
+## TODO
+1) Implement tests
+2) Keep trace of timestamp of last poll for use in the where statement
```

Comment:
This seems like an important piece to figure out.  Is there a pattern that can be used without us passing a time parameter?  Perhaps just suggesting that queries contain something equivalent to `WHERE time > now() - 5s` added to them.